### ISSUE TITLE
MongoDB 6.0 contains breaking changes that make the setup action fail

### ISSUE THREAD

**Issue by tmasternak** [supercharge/mongodb-github-action#34]
Following the deprecation warning:

>> Warning: the "mongo" shell has been superseded by "mongosh",
which delivers improved usability and compatibility.The "mongo" shell has been deprecated and will be removed in

the latest MongoDB i.e. `6.0` does not support `mongo` as the command line tool which results in the following container startup error:

>> OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "mongo": executable file not found in $PATH: unknown

This affects all users who do not specify a concrete MongoDB version, as `latest` is the default version for the `mongodb-version` parameter.

//cc: @danielmarbach

**Comment by tmasternak** [comment]
@marcuspoehls thank you for the quick response. I see that @ecarruda already started the fix.

**Comment by marcuspoehls** [comment]
@tmasternak Yep, Evandro submitted a pull request. The tests are failing because the MongoDB client (`mongo` vs. `mongosh`) isn’t resolved properly. Tomasz, maybe you have an idea what’s causing the issue?
